_overviews/overview/scala-tools.md

```diff
@@ -0,0 +1,320 @@
+
+
+### Creating a first build.sbt file
+
+At this point you only need two more things to run a “Hello, world” project:
+
+- A *build.sbt* file
+- A *Hello.scala* file
+
+For a little project like this, the *build.sbt* file only needs to contain a few lines, like this:
+
+```scala
+name := "HelloWorld"
+version := "1.0"
+scalaVersion := "{{ site.scala-version }}"
+```
+
+Because sbt projects use a standard directory structure, sbt will be able to find everything else it needs.
+
+Now you just need to add a little “Hello, world” program.
+
+
+### A “Hello, world” program
+
+In large projects, all of your Scala source code files will go under the *src/main/scala* and *src/test/scala* directories, but for a little sample project like this, you can put your source code file in the root directory of your project. Therefore, create a file named *HelloWorld.scala* in the root directory with these contents:
+
+```scala
+@main def helloWorld = println("Hello, world")
+```
+
+Now, use the `sbt run` command to compile and run your project. When you do so, you’ll see output that looks like this:
+
+```bash
+$ sbt run
+
+[info] welcome to sbt
+[info] loading settings for project ...
+[info] loading project definition
+[info] loading settings for project root from build.sbt ...
+[info] Compiling 1 Scala source ...
+[info] running helloWorld 
+Hello, world
+[success] Total time: 4 s
+```
+
+The first time you run `sbt` it downloads everything it needs, and that can take a little while to run, but after that it gets much faster. 
+
+Also, once you get this first step working, you’ll find that it’s much faster to run sbt interactively. To do that, first run the `sbt` command by itself:
+
+```bash
+$ sbt
+
+[info] welcome to sbt
+[info] loading settings for project ...
+[info] loading project definition ...
+[info] loading settings for project root from build.sbt ...
+[info] sbt server started at local:///Users/al/.sbt/1.0/server/7d26bae825c38a31074c/sock
+sbt:hello-world> _
+```
+
+Then inside this sbt shell, execute its `run` command:
+
+````
+sbt:hello-world> run
+
+[info] running helloWorld 
+Hello, world
+[success] Total time: 0 s
+````
+
+There, that’s much faster.
+
+If you type `help` at the sbt command prompt you’ll see a list of other commands you can run. But for now, just type `exit` to leave the sbt shell. (You can also press `CTRL-D` instead of typing `exit`.)
+
+
+### Other build tools for Scala
+
+<!-- TODO: add Fury? -->
+While sbt is widely used, there are other tools you can use to build Scala projects:
+
+- [Ant](https://ant.apache.org/)
+- [Gradle](https://gradle.org/)
+- [Maven](https://maven.apache.org/)
+- [Mill](https://www.lihaoyi.com/mill/)
```

Comment:
Maybe it is worth mentioning coursier as a tool for dependency management as well?

Reply:
I added a paragraph for this.